readme.md

```diff
@@ -97,14 +98,65 @@ You can define a custom image and tag in the constructor:
 ### JUnit 4
 
     @Rule
-    final public @NotNull HiveMQEnterpriseTestContainerRule rule 
-        = new HiveMQEnterpriseTestContainerRule("hivemq/hivemq-ce", "2020.2");
+    final public @NotNull HiveMQTestContainerRule rule 
+        = new HiveMQTestContainerRule("hivemq/hivemq-ce", "2020.2");
 
 ### JUnit 5
 
     @RegisterExtension
-    final public @NotNull HiveMQEnterpriseTestContainerExtension extension 
-        = new HiveMQEnterpriseTestContainerExtension("hivemq/hivemq-ce", "2020.2");
+    final public @NotNull HiveMQTestContainerExtension extension 
+        = new HiveMQTestContainerExtension("hivemq/hivemq-ce", "2020.2");
+        
+## Wait Strategy
+
+Per default the HiveMQ Testcontainer waits for the HiveMQ startup log message, but you can define additional
+wait conditions for your HiveMQ Extensions:
+
+### JUnit 4
+
+    @Rule
+    public final @NotNull HiveMQTestContainerRule rule =
+            new HiveMQTestContainerRule()
+                .withExtension(new File("src/test/resources/modifier-extension"))
+                .waitForExtension("My Extension Name");
+
+### JUnit 5
+
+    @RegisterExtension
+    public final @NotNull HiveMQTestContainerExtension extension =
+            new HiveMQTestContainerExtension()
+                .withExtension(new File("src/test/resources/modifier-extension"))
+                .waitForExtension("My Extension Name");
+                
+### JUnit 4
+
+    private final @NotNull HiveMQExtension hiveMQExtension = HiveMQExtension.builder()
+        .id("extension-1")
+        .name("my-extension")
+        .version("1.0")
+        .disabledOnStartup(true)
```

Comment:
does it make sense to wait for an extension that is disabled during startup?

Reply:
no ofc